tests/test_common.js

```diff
@@ -453,3 +453,103 @@ test('percentEncode encodes extra reserved characters', t => {
   t.equal(common.percentEncode('foo+(*)!'), 'foo%2B%28%2A%29%21')
   t.done()
 })
+
+test('formatQueryValue formats values with type "number"', t => {
+  t.deepEqual(common.formatQueryValue(0, 0), [0, '0'])
+  t.deepEqual(common.formatQueryValue(0, -1000), [0, '-1000'])
+  t.deepEqual(common.formatQueryValue(0, 500), [0, '500'])
+  t.done()
+})
+
+test('formatQueryValue formats null & undefined values', t => {
+  t.deepEqual(common.formatQueryValue(0, null), [0, ''])
+  t.deepEqual(common.formatQueryValue(0, undefined), [0, ''])
+  t.done()
+})
+
+test('formatQueryValue formats values with type "string"', t => {
+  // when stringFormattingFn is not passed
+  t.deepEqual(common.formatQueryValue(0, 'foo'), [0, 'foo'])
+
+  // when stringFormattingFn is passed
```

Comment:
We've been working on making the unit size of tests as small as possible, so I'd probably break this up into two separate tests.

Reply:
Addressed in https://github.com/nock/nock/pull/1598/commits/37196f142c211708611e5b6a6e1e33255acf5681